src/ontology/mondo-ingest.Makefile

```diff
@@ -464,16 +464,6 @@ matches: lexical-matches extract-unmapped-matches combine-unmapped-exact-lexmatc
 #############################
 ###### Slurp pipeline #######
 #############################
-# todo: What if the mirror was out of date, but there's already a .db file there, but it's not up to date? will it force update?
-$(COMPONENTSDIR)/%.db: $(COMPONENTSDIR)/%.owl
```

Comment:
## Delete: `$(COMPONENTSDIR)/%.db`
It is subsumed by new `%.db` goal. The steps of the goal were the exact same.